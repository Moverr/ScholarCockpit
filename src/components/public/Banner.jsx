import React  from 'react'
import { logTable } from '../../utils/Logger';

 

function Banner() {
 
  return (
    <div  className="banner">
      <h1 > SCHOOL MANAGEMENT SYSTEM <br/> v1.0 beta</h1>
    </div>

  )

}
export default Banner;