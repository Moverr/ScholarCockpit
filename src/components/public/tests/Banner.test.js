import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Banner from '../Banner';





let wrapper = shallow(<Banner />);



describe('Testing Input FIeld', () => {


  it('should  render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Should have a class banner', () => {
    expect(wrapper).toMatchSnapshot();
  });
})



