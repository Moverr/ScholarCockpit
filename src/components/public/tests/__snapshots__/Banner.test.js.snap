// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input FIeld Should have a class banner 1`] = `
<div
  className="banner"
>
  <h1>
     SCHOOL MANAGEMENT SYSTEM 
    <br />
     v1.0 beta
  </h1>
</div>
`;

exports[`Testing Input FIeld should  render correctly 1`] = `
<div
  className="banner"
>
  <h1>
     SCHOOL MANAGEMENT SYSTEM 
    <br />
     v1.0 beta
  </h1>
</div>
`;
